apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -26,11 +26,9 @@ struct ChatView: View {
     @State private var connectionStats: ConnectionStats?
     @State private var customUserProfile: Profile?
     @State private var connectionCode: String?
-    @State private var tableView: UITableView?
     @State private var loadingItems = false
     @State private var firstPage = false
-    @State private var itemsInView: Set<String> = []
```

Comment:
Constantly updating the itemsInView state while scrolling,
will cause the whole `ChatView` to be re-evaluated every time new chatItem appears